src/Compiler/CompilerUtils.cpp

```diff
@@ -239,19 +242,21 @@ struct Command {
                    << "Program path: " << _path << "\n"
                    << "Command execution failed."
                    << "\n";
-      exit(rc);
+      return rc;
     }
 
     // Restore saved work directory.
     llvm::sys::fs::set_current_path(cur_wdir);
+    return 0;
```

Comment:
return CompilerSuccess;

Reply:
I can, but error code are both from execution and compilation. Keeping 0 is fine I believe.

Reply:
ok